src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ParagraphSmall } from '../typography';
+import { useTheme } from '../../providers';
+import { StyledBanner, StyledContainer } from './styles';
+import { CheckIcon, ClearIcon, InfoIcon, WarningIcon } from '../icons';
+import { BannerType, BannerProps } from './types';
+import { Block } from 'baseui/block';
+
+export const Banner = ({ type = BannerType.info, text }: BannerProps) => {
+  const {
+    theme: {
+      current: {
+        colors: { primary },
+        customColors: { red0, green0, yellow0 },
+        sizing: { scale0 },
+      },
+    },
+  } = useTheme();
+
+  const getBannerBackgroundColor = () => {
+    const map = {
+      [BannerType.success]: green0,
+      [BannerType.error]: red0,
+      [BannerType.warning]: yellow0,
```

Comment:
I think this in yellow5 in the design

Reply:
Oh you're right, I just assumed it was the same haha